polling_stations/apps/data_collection/management/commands/import_newham.py

```diff
@@ -3,35 +3,45 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E09000025"
-    addresses_name = "local.2018-05-03/Version 2/LBNewham Democracy_Club__03May2018.TSV"
-    stations_name = "local.2018-05-03/Version 2/LBNewham Democracy_Club__03May2018.TSV"
-    elections = ["local.2018-05-03", "mayor.newham.2018-05-03"]
+    addresses_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Newham.tsv"
+    stations_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019Newham.tsv"
+    elections = ["europarl.2019-05-23"]
     csv_delimiter = "\t"
 
-    def address_record_to_dict(self, record):
-
-        if record.addressline6 == "E16 1EF":
-            return None
-
-        if record.property_urn == "10090852604":
-            return None
-
-        if record.property_urn == "10034510101":
-            rec = super().address_record_to_dict(record)
-            rec["postcode"] = "E13 8NA"
-            return rec
+    def station_record_to_dict(self, record):
+        # Fix from: 890908:polling_stations/apps/data_collection/management/commands/misc_fixes.py:224
+        if record.polling_place_id == "4483":
+            record = record._replace(polling_place_easting="-0.003917")
+            record = record._replace(polling_place_northing="51.540183")
```

Comment:
This will put you in the sea just off the Isles of Scilly and throw `WARNING: Polling station 4483 is not covered by any council area - manual check recommended`

If you're going to pass in an `polling_place_easting` and `polling_place_northing`, they need to actually be eastings and northings.

If you want to use lat/long, this needs to be:

```py
if record.polling_place_id == "4483":
    rec = super().station_record_to_dict(record)
    rec["location"] = Point(-0.003917, 51.540183, srid=4326)
    return rec
```